README.md

```diff
@@ -277,15 +277,17 @@ See also the [How To Page](https://ddnexus.github.io/pagy/docs/how-to)
 
 ### SupeRails
 
-[<img src="https://img.youtube.com/vi/1tsWL4EjhMo/0.jpg" width="360">](https://www.youtube.com/watch?v=1tsWL4EjhMo)
+[<img src="https://img.youtube.com/vi/1tsWL4EjhMo/0.jpg" width="360" title="15 min - Beginner friendly - Shows installation and use of some pagy extras">](https://www.youtube.com/watch?v=1tsWL4EjhMo)
 
-[<img src="https://img.youtube.com/vi/ScxUqW29F7E/0.jpg" width="360">](https://www.youtube.com/watch?v=ScxUqW29F7E)
+[<img src="https://img.youtube.com/vi/ScxUqW29F7E/0.jpg" width="360" title="18 min - Intermediate Skill Level - 'Load More' pagination using Turbo Streams">](https://www.youtube.com/watch?v=ScxUqW29F7E)
 
-[<img src="https://img.youtube.com/vi/A9q6YwhLCyI/0.jpg" width="360">](https://www.youtube.com/watch?v=A9q6YwhLCyI)
+[<img src="https://img.youtube.com/vi/A9q6YwhLCyI/0.jpg" title="17 min - Intermediate Skill Level - Pagination with Search (Ransack) and Hotwire + Infinite (Countless) Pagination" width="360">](https://www.youtube.com/watch?v=A9q6YwhLCyI)
+
+[<img src="https://img.youtube.com/vi/A9q6YwhLCyI/0.jpg" title="12:52 min - Intermediate Skill Level - API based pagination + using pagy_metadata" width="360">](https://www.youtube.com/watch?v=Qoq6HZ8gdDE)
```

Comment:
these look like the same "src"

Reply:
Resolved with: [b6cb7f9e](https://github.com/ddnexus/pagy/pull/510/commits/b6cb7f9e64e94a8b3e3b61fc83683529f6118c69)